src/admin-views/settings/tabs/display/display-calendar.php

```diff
@@ -30,6 +33,13 @@
 	'default' => esc_html__( 'Default Page Template', 'the-events-calendar' ),
 ];
 
+$templates = get_page_templates();
+ksort( $templates );
+
+foreach ( array_keys( $templates ) as $template ) {
+	$template_options[ $templates[ $template ] ] = $template;
```

Comment:
`get_page_templates()` returns an array. Here is an example of what it may look like:
```
Array (
	[template-contact.php] => Contact
	[template-landing-page.php] => Landing Page
	[template-another-template.php] => Another Template
)
```

For the `$template_options` array, are we wanting the template name (i.e. "Landing Page", "Another Template") to be the key or the template filename (i.e. "template-landing-page.php", "template-another-template.php"?

Reply:
You could use `array_flip()` instead.